docs(tutorial): VM's with PN and PG on PXE (#344)
scaleway/docs-content: tutorials/load-balancer-private-network/index.mdx

<Message type="requirement">

  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key)
  - You have [two Elastic Metal servers](/compute/instances/how-to/create-an-instance) that run on Promox
  - You have [created a Private Network](/network/vpc/how-to/use-private-networks/) and added your resources to it
  - You have [created a Public Gateway](/network/vpc/how-to/create-a-public-gateway/) with DHCP enabled
  - You have cre
  - You have a domain or subdomain pointed to your Instance

</Message>

## Preparing Proxmox

1. Log into the Proxmox web interface and select your server in the datacenter view. Then click **Network** in the menu.

> 2. Click **Create** > **Linux Bridge**. The configuration wizard displays. Enter private IP for your Elastic Metal server in CIDR format. The **Bridge port** is the name of your primary network interface, followed by a dot and your VLAN-ID. Tick the boxes **Autostart** and **VLAN aware**, then click **Create** to create the bridge.

Reviewer: ```suggestion
2. Click **Create** > **Linux Bridge**. The configuration wizard displays. Enter a private IP for your Elastic Metal server in CIDR format. The **Bridge port** is the name of your primary network interface, followed by a dot and your VLAN-ID. Tick the boxes **Autostart** and **VLAN aware**, then click **Create** to create the bridge.
```

    <Lightbox src="scaleway-px-create-linux-bridge.png" alt="" />

3. Click **Apply configuration** to activate the new configuration.

4. Go to the **Datacenter** view and click **Create Cluster**. 

> 5. Enter a name for the cluster and select your Private Network brdige `vmbr1` as cluster network. Then click **Create** to lauch cluster creation. 

Reviewer: ```suggestion
5. Enter a name for the cluster and select your Private Network bridge `vmbr1` as cluster network. Then click **Create** to launch cluster creation. 
```

6. Select the cluster in the cluster list and click **Join Information**. The cluster join information displays.

7. Click **Copy Information** to copy the cluster join information into the clipboard. 

> 8. Go to the Proxmox interface of your second ElasticMetal server and click **Cluster** to enter the cluster configuration from the Datacenter view of the machine. 

Reviewer: ```suggestion
8. Go to the Proxmox interface of your second Elastic Metal server and click **Cluster** to enter the cluster configuration from the Datacenter view of the machine. 
```

9. Click **Join Cluster** and paste the cluster join information in the form. 

> 10. Enter your first Elastic Metal servers `root` password and click **Join** to link the machine to the cluster. 

Reviewer: ```suggestion
10. Enter your first Elastic Metal server's `root` password and click **Join** to link the machine to the cluster. 
```

> The second node appears now in the cluster in the Datacenter view of Proxmox. You can manage both machines using a single interface. The communication between members of the cluster is entirely in your Private Network, ensuring the highest level of security and the lowest latency. 

Reviewer: ```suggestion
The second node appears now in the cluster in the Datacenter view of Proxmox. You can manage both machines using a single interface. Communication between members of the cluster is entirely via your Private Network, ensuring the highest level of security and the lowest latency. 
```

## Creating Virtual Machines

Proxmox is capable of managing Linux Kernel Virtual Machines (KVM) as well as LXC containers. KVM provides full virtualization for complete Windows and Linux images whilst LXC containers are lightweight and perfect to run conflict-free Linux applications. 
Follow this guide for information how to [configure virtual machines and LXC containers on Proxmox](https://www.scaleway.com/en/docs/tutorials/configure-failover-proxmox/).

> Create or clone your virtual machines on both servers of the cluster to distibute the load between them using a Load Balancer. Your virtual machines can use the Public Gateway and dynamic NAT to communicate with the Internet for downloads and updates. If you want to expose certains ports of your VM's directly, configure [static NAT rules](/network/vpc/how-to/configure-a-public-gateway/#how-to-review-and-configure-nat)) for them. 

Reviewer: ```suggestion
Create or clone your virtual machines on both servers of the cluster to distibute the load between them using a Load Balancer. Your virtual machines can use the Public Gateway and dynamic NAT to communicate with the Internet for downloads and updates. If you want to expose certains ports of your VMs directly, configure [static NAT rules](/network/vpc/how-to/configure-a-public-gateway/#how-to-review-and-configure-nat)) for them. 
```
